src/Card/CardHeader.jsx

```diff
@@ -26,18 +27,26 @@ export default class CardHeader extends Component {
       compose: 'Merge',
     });
 
+    if (typeof title === 'string') {
+      return (
+        <div className={this.createHeaderClassName()}>
+          {actions ? <CardActions {...this.props} /> : null }
+          {title ? <div className={themeStyles.titleWrapper}><span className={themeStyles.title} title={title}>{title}</span></div> : null }
+        </div>
+      );
+    }
     return (
-      <div className={this.createHeaderClassName()}>
-        {actions ? <CardActions {...this.props} /> : null }
-        {title ? <div className={themeStyles.titleWrapper}><span className={themeStyles.title} title={title}>{title}</span></div> : null }
-      </div>
+      <PlaceholderTextComponent {...this.props} />
```

Comment:
The text prop of PlaceHolderTextComponent is a string... the render method of PlaceHolderTextComponent simply replaces any occurrence of a regex with a span with the appropriate css style. so in the end we might have:

`some title <span className={classname}></span>`

I was thinking we'd have something like

`<span title="some calculated title string">some title <span className={classname}></span></span>`

this way, when people hover on it (or screen reader), we have an actual title property

Reply:
totally don't need the <Placeholder> tag here... it's already passed to you... see previous comment around inline if